app/src/main/java/org/p2p/wallet/auth/ui/onboarding/NewOnboardingFragment.kt

```diff
@@ -65,8 +78,41 @@ class NewOnboardingFragment :
         signInHelper.showSignInDialog(requireContext(), googleSignInLauncher)
     }
 
+    override fun showError(error: String) {
+        view?.post {
```

Comment:
requireView()

Reply:
could be crash if there is an a delay in call